projects/kalidea/kaligraphi/src/lib/04-overlay/kal-loader/kal-loader.component.ts

```diff
@@ -15,16 +14,16 @@ export class KalLoaderComponent {
 
   message: string;
 
+  @HostBinding('attr.class')
+  _classes = '';
```

Comment:
private ?

Reply:
can't because hostbinding is accessed from outside this class